src/Test/TestState.php

```diff
@@ -122,7 +123,7 @@ public static function bootFromContainer(ContainerInterface $container): void
         $configuration = new Configuration();
 
         try {
-            $configuration->setManagerRegistry($container->get('doctrine'));
+            $configuration->setManagerRegistry($container->get(ChainManagerRegistry::class));
```

Comment:
This logic is called in bundless-usage - as mentioned above, we'll have to do what the compiler pass does manually.